### ISSUE TITLE
Image Consumer is not accepting absolute path

### ISSUE THREAD

**Issue by PeterAkakpo** [CasparCG/server#1285]
### Expected behaviour
Image consumer should accept a custom file path, when it is not supplier then it should default to the media path.

### Current behaviour
Image consumer saves to only media path and does not accept absolute file names
----

### Steps to reproduce
1. add 1 image "c:\users\peter\desktop\snap.png"

nothing is written to the desktop

### Environment
* Commit: latest build
* Server version: 2.3
* Operating system: Windows 10

---

### Screenshots
If applicable, add screenshots as complementary information.

**Comment by dotarmin** [comment]
Have you tried to escape the backslashes?

**Comment by PeterAkakpo** [comment]
Yes, absolute path is totally ignored. all images goes into the media folder

**Comment by Julusian** [comment]
The code calls the parameter the 'filename' and prepends the media folder, so this is working as intended right now. This could probably be improved, but some thought might need to be put into whether there will be issues with file permissions

**Comment by LRTNZ** [comment]
Knowing what Linux can be like, I feel that it may be where most of the issues will be had when it comes to file permissions, and saving media. How many times would someone using this software on a Windows machine try saving media files to an elevated privilege path?
A random thought I had for a solution, is that maybe the media path section could be changed to:
    `<media>`  
     `<source>Path to source here</source>`  
      `<dest>Path to destination here</dest>` 
    `</media>`

**Comment by PeterAkakpo** [comment]
there are too many restrictions and assumptions in this software and its about time that most of such restrictions are lifted.  standard programming does not allow for such restriction. those assumption could be implemented as default but the user must be allow to change those defaults when they wish to.  casparcg is turning out to be an open source software under lock and key

**Comment by PeterAkakpo** [comment]
its not everybody who wants to keep files in root folders. at least permit sub folders

**Comment by PeterAkakpo** [comment]
> The code calls the parameter the 'filename' and prepends the media folder, so this is working as intended right now. This could probably be improved, but some thought might need to be put into whether there will be issues with file permissions

filename can also mean absolute path
